extract_firmware.py

```diff
@@ -406,14 +410,14 @@ def decompress(prefix, start, s):
 
 if VERSION in ARCHIVE_ORDERS:
     gzip_starts = list(m.start(0) for m in re.finditer(
-        re.escape("\x1f\x8b\x08"), kernel))
+        re.escape(b"\x1f\x8b\x08"), kernel))
     idx = 0
     names = ARCHIVE_ORDERS[VERSION]
-    for start, end in itertools.izip(gzip_starts,
+    for start, end in zip(gzip_starts,
```

Comment:
Please fix up alignment on next line.

Also add a commit description which explains the at_is_offset thing (you put it in the PR, but that's a GH thing, not a git thing).

Reply:
Done. I force pushed a new commit.